### DISCUSSION TITLE
Fatal PHP error and empty HTML upon container creation

### DISCUSSION THREAD

**Discussion by ghxstxch** [firefly-iii/firefly-iii#6239] [Firefly III]
Hey there, running into an issue which is likely attributed to my lack of knowledge with Portainer and docker-compose. Both the database and firefly app are running according to 'docker ps' and portainer, however, all I get when navigating to it is an empty HTML page. Logs show this: 

> https://pastebin.com/TsAsGkrr

My docker-compose:

> https://pastebin.com/GYUKJwQC

and my .env file: 

> https://pastebin.com/6tTdGaCz

**Comment by ghxstxch** [comment]
I'm not sure why it did this while writing it out, attempting to clean this up

EDIT: fixed the formatting, uploaded them to pastebin. Sorry!

**Reply by JC5** [reply]
Is your `APP_KEY` exactly 32 chars?

**Reply by ghxstxch** [reply]
> Is your `APP_KEY` exactly 32 chars?

This lead me in the correct direction, I forgot to omit '#' and it was commenting itself out. However I am now running into a separate issue, would you like me to open a separate Q/A? 

It refers to "Could not poll the database: SQLSTATE[HY000] [1045] Access denied for user 'fireflyadmin'@'192.168.16.3' (using password: YES) (SQL: select `id`, `name`, `data` from `configuration` where `name` = is_demo_site and `configuration`.`deleted_at` is null limit 1) "

**Reply by JC5** [reply]
This is pretty common and usually happens when you change the database password after the container has run even once. Or when the password is not equal on both sides.

**Reply by ghxstxch** [reply]
I apologize for the late response - I have used the same password throughout, and both are equal

**Reply by JC5** [reply]
Sorry, I'm running out of ideas.
